PWGCF/FEMTOSCOPY/FemtoDream/AliFemtoDreamTrackHist.h

```diff
@@ -18,7 +18,7 @@
 class AliFemtoDreamTrackHist {
  public:
   AliFemtoDreamTrackHist();
-  AliFemtoDreamTrackHist(bool DCADist, bool CombSig, bool TOFM);
+  AliFemtoDreamTrackHist(bool DCADist, bool CombSig, bool TOFM, float fpTmin = 0, float fpTmax = 0);
```

Comment:
You want to call the argument pTmin and pTmax the f in front is reserved for members.

Reply:
Also, can you as a default the 'old' values 0.5 and 4.05